### DISCUSSION TITLE
表格外的文字被裁剪了，没有看到参数可以保留

### DISCUSSION THREAD

**Discussion by young114** [RapidAI/TableStructureRec#102] [Q&A]
图片90%是表格，但标题和页尾签也很重要，需要保留

**Comment by Joker1212** [comment]
emmm,其实返回信息里面有OCR的结果，只是没有单独返回非表格中的ocr信息，我的建议是首先进行版面识别，再将识别为table的区域截图进行表格识别，因为版面识别能比较准确地得到表格标题，表格footer和其他区域，比直接ocr获取内容可能更准确

**Reply by Joker1212** [reply]
版面识别也可以采用rapid的方案，参考 RapidLayout  https://github.com/RapidAI/RapidLayout

**Reply by young114** [reply]
是同一个图里，90%区域是表格，其他区域和表格内容都需要，想过说采用两个模型一起跑，但觉得太麻烦了

**Reply by Joker1212** [reply]
那就执行完了直接使用ocr结果做过滤呢，因为表格的box匹配了一部分ocr的box，填充了文字，其余没匹配上的ocr的信息包含box和文字信息，你可以根据坐标来得到表格上方和下方的内容，且已经排序了

**Reply by Joker1212** [reply]
这个库没有实现的内容是，返回结果的ocr_res包含了匹配到表格中的内容和未匹配到表格中的内容，需要你找出未匹配到的内容，做排序处理，emmm,后续可能会单独把这个返回，也欢迎提pr哈，其实源码已经做了一部分区分了
